test_cycle:
  delta_t: 0.1            # Control loop / timer period in seconds
  time_per_segment: 5.0    # How long to hold each target [s]
  targets: [-1, 1, 10, -20, -3]
  nr_inputs: 1
  nr_outputs: 1

model_lti_system:
  A:
  - [0.4, -0.03, -0.08]
  - [-0.04 , 0.3, -0.06]
  - [-0.08, -1.4, 0.45]
  B:
    - [0.01]
    - [0.03]
    - [0.08]
  C:
    - [-8.19, 0.2, 3.115]
  D:
    - [0.0]


control_system:
  nr_inputs: 1
  nr_outputs: 1
  delta_t: 0.1


pid_controller:
  pid_constants:
    "[-30.0,0.0]":
      kp: 40
      ki: 0
      kd: 0.1
      kdd: 0.0
    "(0.0,20.0)":
      kp: 30
      ki: 0
      kd: 0.15
      kdd: 0.0
    "[20.0,30.0]":
      kp: 10
      ki: 0
      kd: 0.2
      kdd: 0.05
  integral_limit: 100